views/user.py

```diff
@@ -58,3 +58,26 @@ def login():
     return jsonify({
         "success": 1
     })
+
+
+@user_blueprint.route("check_login", methods=["GET"])
+@login_required
+def check_login():
+    return jsonify({
+        "success": 1,
+        "msg": "user: {} sign in successfully.".format(current_user.email)
+    })
+
+
+@user_blueprint.route("logout", methods=["GET"])
+@login_required
+def logout():
+    logout_user()
+    return jsonify({
+        "success": 1
+    })
+
+
+@user_blueprint.route("login_needed", methods=["GET"])
+def login_needed():
+    return "You need login at first."
```

Comment:
Does this return 401 or 200? We'd better return 401 in this case, and keep returning json with a standardized error message.

Plus, if the frontend and backend are completely separated, I guess there will be no traditional redirects; all redirects are just (automatically) calling another view handler in the backend rather than the browser explicitly visiting another URL. I think you should somehow elaborate "you will be redirected to this API" (in the PR message) a little bit: is this redirection a backend-only behaviour that's transparent for the frontend, or something that needs special treatment in the frontend?